modules/researchsoftware/slides-software.md

```diff
@@ -12,66 +12,145 @@ order: 1
 
 <!-- .slide: data-state="standard" -->
 
-## Software is data
-
-Some data enthousiasts might say: *"Software is just data"*
+## Some history
 <center>
-<img src="media/bits_heart.jpg" width="55%">
+<img class="rotate-90" src="media/Lochkarte_Tanzorgel_cutout.jpg"  width="40%" data-rotate="90">
 </center>
+<small>Photo by <a href="https://www.wikidata.org/wiki/Q106975068">Stefan Kuhn</a> on <a href="https://commons.wikimedia.org/wiki/File:Lochkarte_Tanzorgel.jpg">Wikimedia</a>
+</small>
+
+
+Note:
+Who knows what this is?
```

Comment:
This is the musical one, right? Maybe for future instructors it would be good to add a note of what it actually is :)

Reply:
```suggestion
Who knows what this is?

See next slide for explanation.
```